app/src/main/java/com/infomaniak/swisstransfer/ui/navigation/NavigationDestination.kt

```diff
@@ -110,48 +111,96 @@ sealed class MainNavigation : NavigationDestination() {
  * Sealed class representing the navigation arguments for the new transfer flow.
  */
 @Serializable
-sealed class NewTransferNavigation : NavigationDestination() {
+sealed class NewTransferNavigation(override val matomoValue: String) : NavigationDestination() {
 
     @Serializable
-    data object ImportFilesDestination : NewTransferNavigation()
+    data object ImportFilesDestination : NewTransferNavigation("ImportFileView")
 
     @Serializable
-    data class ValidateUserEmailDestination(val authorEmail: String) : NewTransferNavigation()
+    data class ValidateUserEmailDestination(val authorEmail: String) : NewTransferNavigation("ValidateUserEmailView")
 
     @Serializable
     data class UploadProgressDestination(
         val transferType: TransferTypeUi,
         val totalSize: Long,
         val authorEmail: String?,
-    ) : NewTransferNavigation()
+    ) : NewTransferNavigation("UploadProgressView")
 
     @Serializable
     data class UploadSuccessDestination(
         val transferType: TransferTypeUi,
         val transferUuid: String,
         val transferUrl: String,
-    ) : NewTransferNavigation()
+    ) : NewTransferNavigation("UploadSuccessView")
 
     @Serializable
     data class UploadErrorDestination(
         val transferType: TransferTypeUi,
         val totalSize: Long,
         val authorEmail: String?,
-    ) : NewTransferNavigation()
+    ) : NewTransferNavigation("UploadErrorView")
```

Comment:
Maybe this will change as well with @LouisCAD 's PR. What will happen to this tracking then?

Reply:
Same as above, we want to keep matomoValues agnostic from our class name to allow to be iso with iOS